src/pushsource/__init__.py

```diff
@@ -17,9 +17,12 @@
     ContainerImagePullInfo,
     SourceContainerImagePushItem,
     OperatorManifestPushItem,
+    VMIPushItem,
```

Comment:
There is a mismatch in naming scheme here, e.g. VMIPushItem vs AmiPushItem.

I was going to recommend that your classes should be renamed to be consistent with those present, e.g. VmiPushItem.  However, PEP8 says we should "capitalize all the letters of the acronym" so I guess it's the existing classes which are wrong.  Let's stick with the PEP8-consistent names and fix up the others later.